internal/tools/go.mod

```
@@ -1,11 +1,13 @@
 module go.opentelemetry.io/otel/internal/tools
 
-go 1.21
+go 1.22.1
+
+toolchain go1.23.0
```

https://github.com/open-telemetry/opentelemetry-go/issues/5660

```suggestion
go 1.21
```

---

That won't work though, because the linter overrides the go.mod, and causes an unclean repository.